src/test/java/ezvcard/io/roundtrip/XCardRoundTripTest.java

```diff
@@ -0,0 +1,82 @@
+package ezvcard.io.roundtrip;
+
+import java.io.*;
+
+import org.junit.Test;
+
+import ezvcard.VCardVersion;
+import ezvcard.io.StreamReader;
+import ezvcard.io.StreamWriter;
+import ezvcard.io.xml.XCardReader;
+import ezvcard.io.xml.XCardWriter;
+
+public class XCardRoundTripTest extends RoundTripTestBase {
+
+	public XCardRoundTripTest() throws Exception {
+		updateSamples(VCardVersion.V4_0,
+				"outlook-2003" // &#12; in fburl is not valid xml
+		);
+		updateSamples(VCardVersion.V3_0);
+	}
+
+	@Test
+	public void equals_compare_vcard_4_to_xcard() throws Exception {
+		convertAllFromVCard(VCardVersion.V4_0, true, false,
+				"outlook", // newlines not preserved on linux
+				"black_berry", // encoding parameter for image?
+				"rfc2426" // {TYPE=[INTERNET,pref]} != {TYPE=[INTERNET],PREF=[1]} ?
+		);
+	}
+
+	@Test
+	public void content_compare_vcard_4_to_xcard() throws Exception {
+		convertAllFromVCard(VCardVersion.V4_0, false, true,
+				"outlook-2007", // newlines not preserved on linux
+				"android", // empty <type/> tag added to email
+				"ms_outlook", // empty <street/> tag added to home adr
```

Comment:
Looks like the empty tag doesn't appear on the build server either. Here's my java version, just in case:
```
$ java -version
openjdk version "1.8.0_92"
OpenJDK Runtime Environment (build 1.8.0_92-b14)
OpenJDK 64-Bit Server VM (build 25.92-b14, mixed mode)
```